VideoGenerator/library/src/main/kotlin/VideoGeneratorImpl.kt

```diff
@@ -1,70 +1,88 @@
 import org.bytedeco.ffmpeg.global.avcodec
 import org.bytedeco.javacv.FFmpegFrameRecorder
 import org.bytedeco.javacv.Java2DFrameConverter
+import java.awt.image.BufferedImage
```

Comment:
BufferedImage is an awt class which is not available on android as far as previous discussion went about :(

Reply:
Oof. Another round of refactoring for me :)

Reply:
2 hours later: It works again